napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, layer):
         self.grid_layout = QGridLayout(self)
         self.grid_layout.setContentsMargins(0, 0, 0, 0)
         self.grid_layout.setSpacing(2)
-        self.grid_layout.setColumnMinimumWidth(0, 86)
+        self.grid_layout.setColumnMinimumWidth(0, 94)
```

Comment:
Just another symptom of the issues raised in #2470 and #3620, so I'm connecting to those issues for visibility.